content/geoip/docs/web-services/sandbox-environment.mdx

```diff
@@ -0,0 +1,59 @@
+---
+draft: false
+title: Testing GeoIP Web Services in the MaxMind Sandbox
+---
+
+import SandboxIntro from '/content/_sandbox-intro.mdx';
+import SandboxLicenseKey from '/content/_sandbox-license-key.mdx';
+import SandboxLimits from '/content/_sandbox-limits.mdx';
+import SandboxLearnMore from '/content/_sandbox-learn-more.mdx';
+
+<SandboxIntro />
+
+## Generate a License Key for the Sandbox
+
+<SandboxLicenseKey />
+
+## Build your integration
+
+Consult our [quick start guide](/geoip/geolocate-an-ip/web-services?lang=en) or
+our [list of client
+APIs](/geoip/geolocate-an-ip/web-services?lang=en#client-apis) to develop your
+integration.
+
+## Point your integration to the Sandbox service endpoints
+
+Use the service endpoints for the Sandbox version of the web services, as listed
+in the table below:
+
+| Sandbox Service | HTTP Method | Endpoint                                                       |
+| --------------- | ----------- | -------------------------------------------------------------- |
+| Country         | `GET`       | `https://sandbox.maxmind.com/geoip/v2.1/country/{ip_address}`  |
+| City Plus       | `GET`       | `https://sandbox.maxmind.com/geoip/v2.1/city/{ip_address}`     |
+| Insights        | `GET`       | `https://sandbox.maxmind.com/geoip/v2.1/insights/{ip_address}` |
+
+Remember to use the license key generated by your Sandbox account to authenticate your requests.
+
+## Test Data for the GeoIP Sandbox
+
+<Alert type="info">
+The Sandbox will return geolocation and IP intelligence data, but this data is
+not accurate and should not be used to evaluate the efficacy of the minFraud
+scoring engine, or for geolocation or IP intelligence. It is designed for
+testing purposes only.
+</Alert>
+
+There are a limited number of IP addresses in the test database for the GeoIP
+web services on the Sandbox. The Sandbox versions of the GeoIP web services use
+the same data as our test databases, which are published to Github. [View our
+test data on Github.](https://github.com/maxmind/MaxMind-DB/blob/main/source-data/GeoIP2-Precision-Enterprise-Test.json)
+
```

Comment:
FWIW, I tested out all the 3 endpoints on lines 31 to 33 using the file mentioned on line 49, and using the first IP in it, both city and insights returned correct info. For the country one, I got {"code":"IP_ADDRESS_NOT_FOUND","error":"The IP address '::17c:d500' is not in our database."

The file for country is https://github.com/maxmind/MaxMind-DB/blob/main/source-data/GeoIP2-Country-Test.json